doc/extract_swagger.py

```diff
@@ -8,21 +10,21 @@
 from tribler_core.restapi.rest_manager import RESTManager
 
 
-async def start_api():
+async def extract_swagger(destination_fn):
     session = Mock()
     session.config.get_http_api_key = lambda: 'apikey'
     session.config.get_http_api_port = lambda: 8085
     api_manager = RESTManager(session)
     await api_manager.start()
 
+    fp = StringIO()
+    proto = aiohttp.web_protocol.RequestHandler(api_manager.site._runner._server, loop=get_event_loop())
+    proto.connection_made(Mock(is_closing=lambda: False, write=lambda b: fp.write(b.decode())))
+    proto.data_received(b'GET /docs/swagger.json HTTP/1.1\r\n'
```

Comment:
I looked into alternatives for sending the HTTP plain text on this line:

```python
from io import BytesIO
from http.client import HTTPConnection

output = BytesIO()
con = HTTPConnection('localhost')
con.send = output.write
con.putrequest('GET', '/docs/swagger.json', True, True)
con.putheader('Connection', 'close')
con.endheaders()
proto.data_received(output.getvalue())
```

Though I'm not a fan of plain text HTTP GET requests, given the alternative, I think a plain text HTTP message is acceptable here. @devos50 your thoughts?